modules/packages/RangeChunk.chpl

```diff
@@ -84,6 +84,43 @@ module RangeChunk {
       else start..end;
   }
 
+  /*
+    Iterates through a range ``r``, which is blocked up in repeating ```numInds```
+    blocks of size ```blockSize```. Blocks are indexed from 0..numInds-1. 
+    Emits block at index ```idx``` in a cyclic manner.
+  */
+  iter blockCyclicChunks(r: range(?t, boundedType=BoundedRangeType.bounded, ?strided), blockSize: integral, idx: integral, numInds: integral){
```

Comment:
We like to try to keep lines to less than 80 characters, so it would be good to split this into multiple lines. Same for other lines that go above 80 characters.

There is no space between the parenthesis and curly bracket at the end of this line. Please add one.